blackbox/docs/admin/privileges.rst

```diff
@@ -56,11 +76,24 @@ and ``DELETE`` statements, on the object for which the privilege applies.
 .......
 
 Granting ``Data Definition Language (DDL)`` privilege to a user, indicates that
-this user is allowed to execute ``CREATE TABLE``, ``DROP TABLE``,
-``CREATE VIEW``, ``DROP VIEW``, ``CREATE FUNCTION``, ``DROP FUNCTION``,
-``CREATE REPOSITORY``, ``DROP REPOSITORY``, ``CREATE SNAPSHOT``,
-``DROP SNAPSHOT``, ``RESTORE SNAPSHOT``, and ``ALTER`` statements, on the
-object for which the privilege applies.
+this user is allowed to execute the following statements on objects for which
+the privilege applies:
+
+- ``CREATE TABLE``
+- ``DROP TABLE``
+- ``CREATE VIEW``
+- ``DROP VIEW``
+- ``CREATE FUNCTION``
+- ``DROP FUNCTION``
+- ``CREATE REPOSITORY``
+- ``DROP REPOSITORY``
+- ``CREATE SNAPSHOT``
+- ``DROP SNAPSHOT``
+- ``RESTORE SNAPSHOT``
+- ``ALTER``
+- ``CREATE USER`` (Cluster)
+- ``DROP USER``, for users the user created. The superuser can remove any user.
```

Comment:
Not sure if we should have this restriction. Implementation would be simpler if we didn't add it. (As we currently don't persist who created a user)

Reply:
Should they also be allowed to change passwords of existing users?